project/Base.scala

```diff
@@ -47,7 +47,8 @@ class Base extends Build {
     organization := "io.buoyant",
     version := Git.version,
     homepage := Some(url("https://linkerd.io")),
-    scalaVersion in GlobalScope := "2.11.7",
+    scalaVersion in GlobalScope := "2.12.1",
```

Comment:
Thanks!  Even though it might not be strictly necessary, I think we're going to mirror the scala version that finagle uses for now.